ietf/doc/views_status_change.py

```diff
@@ -685,7 +685,7 @@ def last_call(request, name):
     form = LastCallTextForm(initial=dict(last_call_text=escape(last_call_event.text)))
 
     if request.method == 'POST':
-        if "save_last_call_text" in request.POST or "send_last_call_request" in request.POST:
+        if "save_last_call_text" in request.POST or "send_last_call_request" in request.POST and status_change.ad is not None:
```

Comment:
Parentheses might be in order here - this is `if A or (B and C):` I think, which makes sense but it'd help readability to make the intent more explicit